### ISSUE TITLE
MD013: sentence reflow turns literal CJK emphasis delimiters into a strong node

### ISSUE THREAD

**Issue by ziyuEngineer** [rvben/rumdl#879]
## Bug

With only MD013 enabled, sentence reflow changes literal Markdown delimiter
text into a real strong-emphasis node. Preserving non-whitespace characters
is not sufficient to preserve the document's inline structure.

Tested with the official Windows x86_64 binary of **v0.2.73**.
The before/after parse was independently checked with markdown-it's
`commonmark` preset.

## Reproduction

`.rumdl.toml`:

```toml
[global]
enable = ["MD013"]

[MD013]
line-length = 0
reflow = true
reflow-mode = "sentence-per-line"
```

`test.md`:

```markdown
1. **已经完成。**继续执行。
```

```console
rumdl fmt --config .rumdl.toml --no-cache test.md
```

Actual output:

```markdown
1. **已经完成。**
   继续执行。
```

Before:

```html
<ol>
<li>**已经完成。**继续执行。</li>
</ol>
```

After:

```html
<ol>
<li><strong>已经完成。</strong>
继续执行。</li>
</ol>
```

In the input, the punctuation-sensitive delimiter run does not close a
CommonMark strong span, so the asterisks are literal text. Placing whitespace
after it makes it a closing delimiter. MD013 has thereby created formatting
and removed visible asterisks, although no non-whitespace source characters
were changed.

## Expected

Preserve the inline parse. Leaving this paragraph unchanged is preferable
to silently interpreting it as intended bold text. If repairing such
markup is desired, that should be a separate explicit rule, not an incidental
effect of line reflow.

## Controls / acceptance

- A valid strong span such as `1. **已经完成。** 继续执行。` may still reflow
  without losing its original strong node.
- Plain delimiter-like text such as `name__Important.__Next` must stay plain.
- Cover single, double, triple and mixed markers, with CJK and ASCII
  punctuation, and validate tokens/literal text rather than stripping all
  whitespace from the source.
- The reported output is stable on the second pass, so idempotency alone is
  not an adequate guard.

Related #816 and #811 concern recognizing sentence endings in **valid**
spans. Here the input has no strong node; the issue is creating one.

The current main source at
`1c7aabeae1ff3295c4d9b16fd99e9605dd6221dd` has a `preserves_content` guard
based on non-whitespace characters and retained word boundaries in
`src/utils/text_reflow.rs`. That appears insufficient for this case; I have
not run a main binary.

**Comment by rvben** [comment]
Thanks @ziyuEngineer!

Fixed in [06ba814f](https://github.com/rvben/rumdl/commit/06ba814fd11e928184a2e509fbec662057faf729), released in [v0.2.74](https://github.com/rvben/rumdl/releases/tag/v0.2.74). CJK sentence reflow now avoids breaks that would turn literal delimiter text into emphasis.

Closing as completed.
